src/forwardproblem.cpp

```diff
@@ -156,13 +217,9 @@ void ForwardProblem::handleEvent(
     /* store Heaviside information at event occurrence */
     model->froot(t_, x_, dx_, rootvals_);
 
-    /* store timepoint at which the event occurred*/
+    /* store timepoint at which the event occurred */
     discs_.push_back(t_);
 
-    /* extract and store which events occurred */
```

Comment:
Why was this removed?

Reply:
I lifted that up into `ForwardProblem::workForwardProblem`. This is the only place where `handleEvent` is called with `!seflag && !initial_event`. We need it out there, because we need to modify `roots_found_` *after* calling `getRootInfo`.